lib/teleterm/cmd/db.go

```diff
@@ -71,6 +71,7 @@ func newDBCLICommandWithExecer(ctx context.Context, cluster *clusters.Cluster, g
 		dbcmd.WithNoTLS(),
 		dbcmd.WithTolerateMissingCLIClient(),
 		dbcmd.WithExecer(execer),
+		dbcmd.WithOracleOpts(true, true),
```

Comment:
I wish we had a single method for each option.
Given that we don't, can we add them or at least embed comments on what each param means?

Reply:
Done.